### DISCUSSION TITLE
Linkage of config files

### DISCUSSION THREAD

**Discussion by worldpeace-germany** [missionpinball/missionpinball#53] [Q&A] [answered]
Hello,

in https://docs.missionpinball.org/en/latest/hardware/opp/cobrapin/index.html you find an example config file at the very bottom, where it says

```
#One giant config file can get difficult to manage. You can put any of
#  these config sections in its own yaml file and link to it with the
#  config section here
config:
  #- switches_config.yaml
  #- lights_config.yaml
  #- coils_config.yaml
  # ...
```

I would like to know how to do that linkage, but was not able to find that information. Can someone give me a pointer where I can find that information? I am sure it is somewhere in the documentation :-)

Thanks and happy new year to those who enter a new year soon.

**Comment by ralfit0** [comment, marked as answer]
[worldpeace](https://github.com/worldpeace-germany),
You simply un-comment these lines, that's all. Or, more desriptive:
In the same config folder where your config.yaml is located, create another file e.g. switches_config.yaml .
In your config.yaml add these lines:
```
config:
  - switches_config.yaml

```
and it looks for that file.

**Reply by worldpeace-germany** [reply]
Thanks, too easy ;-)
